### PR TITLE
[Find Forms] Fix Axe error

### PR DESCRIPTION
**Pull request by ncksllvn**
## Description
This PR fixes a 508 error with the `/find-forms` page, which is happening now because the page was just recently merged. We have a ticket to write a comprehensive E2E test on this page/widget.

## Testing done
This PR _is_ the test 😉 

## Screenshots
image

## Acceptance criteria
- [ ] 508 issue is resolved

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/find-va-forms/containers/SearchForm.jsx [hunk 1]
```diff
@@ -65,7 +65,7 @@ export class SearchForm extends Component {
           <div className="vads-u-margin-right--2 vads-u-flex--1">
             <input
               className="usa-input vads-u-max-width--100 vads-u-width--full"
-              name="va-form-query"
+              id="va-form-query"
```
Comment (ncksllvn) [inline src/applications/find-va-forms/containers/SearchForm.jsx]: I for some reason though that the the label `for` attribute could work with the `input` name.

**Review by micahchiang — APPROVED** [review]
ltgm

**Review by LinnJS — APPROVED** [review]

**Review by cvalarida — APPROVED** [review]
Can you help me understand how the bug got merged in if it was breaking CI?

**Comment by ncksllvn** [comment]
@cvalarida This is a widget, so it was never rendered in CI until the corresponding page was published in Drupal, so then the page was evaluated during accessibility tests and failed

**Review by rianfowler — APPROVED** [review]

**Pull request merged**